src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -68,6 +68,9 @@ public class Checker extends AbstractAutomaticBean implements MessageDispatcher,
     /** The extension separator. */
     private static final String EXTENSION_SEPARATOR = ".";
 
+    /** Separator to use in string concatenation. */
+    private static final String WHITESPACE_SEPARATOR = " ";
```

Comment:
```suggestion
    private static final String SINGLE_SPACE = " ";
```
We have no need to be ambiguous here. "Whitespace" could be multiple spaces, tabs, newlines, carriage returns, form feeds, vertical tabs, etc (or any combination thereof).

Reply:
I have originally used var `SINGLE_WHITESPACE` but later changed it after review https://github.com/checkstyle/checkstyle/pull/14652#discussion_r1524076122.

Done now.... re-changed it after re-review 😄

Reply:
I have originally used var `SINGLE_WHITESPACE` but later changed it after review https://github.com/checkstyle/checkstyle/pull/14652#discussion_r1524076122.

Done now.... re-changed it after re-review 😄

Reply:
If we do https://github.com/checkstyle/checkstyle/pull/14652#discussion_r1526999404 this constant is not required at all

Reply:
Yep, please do this @MANISH-K-07

Reply:
Sure.. already on it :)